pages/portfolio.html

```diff
@@ -1,12 +1,35 @@
+<!-- must include information about several projects you've completed at Ada (or elsewhere) with links to the GitHub repo if available, descriptions, images, etc. -->
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Alma's portfolio</title>
+  <link href="styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
+<body class="container">
+  <section>
+    <header>
+    <h1 class="main-title">Projects</h1></header>
+    <blockquote>
+      <header><h4 class="viewing-title">viewing party</h4></header>
+      Description: The solution to streaming movies with your friends.
+      <nav>
+        <a href="https://github.com/abecerrilsalas/viewing-party">Click here to view GitHub repo</a></nav>
+        <h4>Solar System</h4>
+        <nav>
+        Description: The solution to organizing solar systems  
+        <a href="https://github.com/tt-ht/solar-system-api">Click here to view GitHub repo</a></nav>
+        <h4>Retro Video Store</h4>
+        <nav>
+        Description:For when you're feeling nostalgic of video stores.
+        <a href="https://github.com/goblinbrooke/retro-video-store">Click here to view GitHub repo</a>
+      </nav>
+      <footer>
+        <p class="copyright">© 2021 Becerril Salas</p>
+      </footer>
+    </blockquote>
```

Comment:
This section isn't actually a quotation, so some tag other than `blockquote` may have been more appropriate here.